lib/usd/utils/CMakeLists.txt

```diff
@@ -84,3 +90,87 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
             DESTINATION ${CMAKE_INSTALL_PREFIX}/lib OPTIONAL
     )
 endif()
+
+# -----------------------------------------------------------------------------
+# sources
+# -----------------------------------------------------------------------------
+
+add_library(${MAYAUSDUTILS_PYTHON_LIBRARY_NAME} SHARED)
+
+target_sources(${MAYAUSDUTILS_PYTHON_LIBRARY_NAME}
+PRIVATE
+  module.cpp
+  wrapMayaTransformAPI.cpp
```

Comment:
@robthebloke  wrapMayaTransformAPI has a dependency to **boost python** and you need to tell CMake to find it's module.

Add below after `add_library(${MAYAUSDUTILS_PYTHON_LIBRARY_NAME} SHARED)`

```
#------------------------------------------------------------------------------
# modules and definitions
#------------------------------------------------------------------------------
if(NOT DEFINED BOOST_ROOT)
    set(BOOST_ROOT ${PXR_USD_LOCATION})
endif()
if(MAYAUSD_DEFINE_BOOST_DEBUG_PYTHON_FLAG)
    set(Boost_USE_DEBUG_PYTHON ON)
endif()
find_package(Boost COMPONENTS python REQUIRED)
```